ui/client/stylesheets/graph.styl

```diff
@@ -496,6 +513,15 @@ espPanelButton()
   top: -25px
   float: right
 
+.row-ace-editor
```

Comment:
do we still need these changes?

Reply:
In first attempt I used AceEditor for type completion and I corrected style for this component. I can move it to separate commit